### PR TITLE
Feature Request: Add actions in ReCaptchaV3 widget

### LINKED ISSUE #209: Adding actions in ReCaptchaV3 widget
Hello, 
I would like at change the default [action](https://developers.google.com/recaptcha/docs/v3#actions) for reCAPTCHA v3.  Default actions is 
` grecaptcha.execute('{{ public_key }}', {action: 'form'})`
I made the following changes. Added additional parameter action to  `ReCaptchaV3` widget and  passing it to context
```
class ReCaptchaV3(ReCaptchaBase):
    template_name = "captcha/widget_v3.html"

    def __init__(self, api_params=None, action="form", *args, **kwargs):
        super(ReCaptchaV3, self).__init__(
            api_params=api_params, *args, **kwargs
        )
        self.action = action

    def get_context(self, name, value, attrs):
        context = super(ReCaptchaV3, self).get_context(name, value, attrs)
        context.update({'action': self.action})
        return context
```
and js_v3.html
` grecaptcha.execute('{{ public_key }}', {action: '{{action}}'})`

It's working fine for me. I don't have much data in admin panel yet to tell more about actions. Can you please add similar functionality or shall I make a pull request if you think this changes wont make any issues?

### PR DESCRIPTION
**Pull request by VenturaFranklin**
Closes #209 @binoyudayan did all of the work but I'm assuming was waiting for a response on the issue before creating a PR.
This would be a nice feature providing [action support](https://developers.google.com/recaptcha/docs/v3#actions)

Which from the docs has the benefits of: 

> - A detailed break-down of data for your top ten actions in the admin console
> - Adaptive risk analysis based on the context of the action, because abusive behavior can vary.

### CODE REVIEW

**Comment by cbracco** [comment]
Hi @thibaudcolas! Is it possible to get any futher movement on this one? Our team would find this enhancement useful, we are using reCAPTCHA on a few different forms and it would be nice to be able to distinguish between them in the reCAPTCHA admin data for analysis purposes.

**Comment by Andrew-Chen-Wang** [comment]
there needs to be a small update for this to continue; I don't know what I'm reading as I haven't used recaptcha in awhile, but if you approve of this, i can merge.

The only requirement I need to know is whether this is backwards compatible,

**Comment by cbracco** [comment]
@Andrew-Chen-Wang I’m not the person who opened this PR, but if you read [the related issue](https://github.com/torchbox/django-recaptcha/issues/209), it’s requesting the ability to customize the `action` value for each reCAPTCHA instance in your codebase. Right now it’s hard-coded as `'form'` and cannot be customized.

This is useful if you have reCAPTCHA on more than one form on your website/app, and you [need to distinguish between the forms in the analytics data provided in the reCAPTCHA admin](https://developers.google.com/recaptcha/docs/v3#actions).

File: captcha/widgets.py [hunk 1]
```diff
@@ -72,15 +72,16 @@ def build_attrs(self, base_attrs, extra_attrs=None):
 class ReCaptchaV3(ReCaptchaBase):
     template_name = "captcha/widget_v3.html"
 
-    def __init__(self, api_params=None, *args, **kwargs):
+    def __init__(self, api_params=None, action="form", *args, **kwargs):
```
Comment (cbracco) [inline captcha/widgets.py:75]: @Andrew-Chen-Wang if the user does not customize the `action` parameter, it should default to `'form'` which is the previously hard-coded value. This piece _should_ be backwards compatible, then?

**Review by Andrew-Chen-Wang — APPROVED** [review]

**Comment by cbracco** [comment]
@Andrew-Chen-Wang the conflict was resolved by my colleague in a new PR: https://github.com/torchbox/django-recaptcha/pull/304 ... Can you please have a look at that one?

**Comment by Stormheg** [comment]
This was completed in #304

**Pull request closed without merging**